export_options.go

```diff
@@ -405,9 +488,29 @@ func addXcode9Properties(exportOpts exportoptions.ExportOptions, teamID, codesig
 	return nil
 }
 
+func addXcode12Properties(exportOpts exportoptions.ExportOptions, distributionBundleIdentifier string) exportoptions.ExportOptions {
+	switch exportOpts.(type) {
+	case exportoptions.AppStoreOptionsModel:
+		options, ok := exportOpts.(exportoptions.AppStoreOptionsModel)
+		if !ok {
```

Comment:
Less important, exportoptions.ExportOptions could have the common fields, so no casting is needed.

Reply:
Unfortunately exportoptions.ExportOptions is an interface not a struct.